From 75d313ba57e13d62888917bd460a01655b264610 Mon Sep 17 00:00:00 2001
From: Tomohiro Komagata <tomohiro.komagata.aj@gmail.com>
Date: Tue, 30 Nov 2021 14:26:54 +0900
Subject: [PATCH 8/8] arm64: dts: renesas: Add support pwm0 for CCPF-SK

pwm0 is used for the fan control via RL78 on the CCPF-SK boards

Signed-off-by: Tomohiro Komagata <tomohiro.komagata.aj@gmail.com>
---
 arch/arm64/boot/dts/renesas/ulcb-ccpf-sk.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/ulcb-ccpf-sk.dtsi b/arch/arm64/boot/dts/renesas/ulcb-ccpf-sk.dtsi
index 04e84b3f9380..7db95110078e 100644
--- a/arch/arm64/boot/dts/renesas/ulcb-ccpf-sk.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb-ccpf-sk.dtsi
@@ -104,6 +104,11 @@ du_pins: du {
 		function = "du";
 	};
 
+	pwm0_pins: pwm0 {
+		groups = "pwm0";
+		function = "pwm0";
+	};
+
 	usb0_pins: usb0 {
 		groups = "usb0";
 		function = "usb0";
@@ -115,6 +120,13 @@ usb30_pins: usb30 {
 	};
 };
 
+&pwm0 {
+	pinctrl-0 = <&pwm0_pins>;
+	pinctrl-names = "default";
+
+	status = "okay";
+};
+
 &usb_extal_clk {
 	clock-frequency = <50000000>;
 };
-- 
2.25.1

